tests/baselines/reference/resolutionCandidateFromPackageJsonField2(moduleresolution=bundler).errors.txt

```diff
@@ -1,11 +1,12 @@
-error TS5095: Option 'bundler' can only be used when 'module' is set to 'preserve' or to 'es2015' or later.
+tsconfig.json(2,5): error TS5095: Option 'bundler' can only be used when 'module' is set to 'preserve' or to 'es2015' or later.
 test.ts(1,19): error TS5097: An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.
 
 
-!!! error TS5095: Option 'bundler' can only be used when 'module' is set to 'preserve' or to 'es2015' or later.
-==== tsconfig.json (0 errors) ====
+==== tsconfig.json (1 errors) ====
     {
         "compilerOptions": {
+        ~~~~~~~~~~~~~~~~~
+!!! error TS5095: Option 'bundler' can only be used when 'module' is set to 'preserve' or to 'es2015' or later.

```

Comment:
This looks like something that would be better reported on the bundler option, except it's not here at all? I guess since it's passed in as a twoslash option, but this does make me wonder if we should be even more specific when possible... (But I do think this PR is better than the status quo anyway)

Reply:
See my comment above